webgallery/models.py

```diff
@@ -19,23 +19,23 @@ def __str__ (self):
       return "%s" % (self.title)
 
    
-class Slider (models.Model):
-   title = models.CharField(max_length = 200)
-   picture = models.ManyToManyField(Picture)
-   
-   def __str__ (self):
-      return "%s" % (self.title)
-   
-   def get_pictures(self):
-      return self.picture.all()
+#class ActivePic (models.Model):
```

Comment:
no comments on what is the purpose of ActivePic, the name is not descriptive please elaborate